src/algebra/lie/classical.lean

```diff
@@ -72,11 +72,9 @@ variables [decidable_eq n] [decidable_eq p] [decidable_eq q] [decidable_eq l]
 variables [comm_ring R]
 
 @[simp] lemma matrix_trace_commutator_zero (X Y : matrix n n R) : matrix.trace n R R ⁅X, Y⁆ = 0 :=
-begin
-  -- TODO: if we use matrix.mul here, we get a timeout
-  change matrix.trace n R R (X * Y - Y * X) = 0,
-  erw [linear_map.map_sub, matrix.trace_mul_comm, sub_self]
-end
+-- TODO: if we use matrix.mul here, we get a timeout
+show matrix.trace n R R (X * Y - Y * X) = 0, by
```

Comment:
Actually, I think that now matrix multiplication can be used, but I will make an amendment when I am back at my computer!